src/styles/ThemeProvider.tsx

```diff
@@ -4,11 +4,61 @@ import { shallowEqual, useSelector } from 'react-redux';
 
 import { selectTheme } from '@/redux/slices/theme';
 
+const getThemeColorMetaTags = (theme) => {
+  switch (theme) {
+    case 'auto': {
+      const themeColorTag1 = document.createElement('meta');
```

Comment:
nit: make variable name more readable. e.g instead of `themeColorTag1`, we can use `themeColorDarkMode`

Reply:
I did your suggestion but I was getting an error when committing _react-func/max-lines-per-function_, which is 30 lines. I modified the code to fit in the 30 lines and in its current state I don't think there is any better way to name them since each can serve a different purpose based on the case.

If you have any other suggestion please share